HybrasylXML/XSD/Status.xsd

```diff
@@ -38,14 +39,10 @@
   </xs:complexType>
 
   <xs:complexType name="CastRestrictions">
-    <xs:sequence minOccurs="0" maxOccurs="1">
-      <xs:element name="Use" type="status:CastRestriction"/>
-      <xs:element name="Receive" type="status:CastRestriction"/>
-    </xs:sequence>
+    <xs:attribute name="Use" type="xs:token"/>
+    <xs:attribute name="Heal" type="xs:token"/>
   </xs:complexType>
-
-  <xs:complexType name="CastRestriction"/>
```

Comment:
If removing CastRestriction as a type, CastRestrictions should be singular.

Reply:
OK.